docker/Dockerfile.dev.api

```diff
@@ -23,4 +23,4 @@ ENV PATH="/venv/bin:${PATH}"
 ENV VIRTUAL_ENV="/venv"
 
 
-ENTRYPOINT ["gunicorn", "keep.api.api:get_app", "--bind" , "0.0.0.0:8080" , "--workers", "1" , "-k" , "uvicorn.workers.UvicornWorker", "-c", "./keep/api/config.py", "--reload"]
+CMD ["gunicorn", "keep.api.api:get_app", "--bind" , "0.0.0.0:8080" , "--workers", "1" , "-k" , "uvicorn.workers.UvicornWorker", "-c", "./keep/api/config.py", "--reload"]
```

Comment:
why?

Reply:
With ENTRYPOINT, the PyCharm debugger can't start the container with custom runtime since ENTRYPOINT is mandatory to execute. CMD allows to overwrite initial command during container start